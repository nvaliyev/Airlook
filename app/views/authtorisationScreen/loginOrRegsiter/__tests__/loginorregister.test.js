import 'react-native';
import React from 'react';
import { act, create } from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import LoginOrRegister from '..';
describe('Login or Register', () => {
    test('Login or Register snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<LoginOrRegister/>);
        expect(snapshot).toMatchSnapshot();
    });
});


