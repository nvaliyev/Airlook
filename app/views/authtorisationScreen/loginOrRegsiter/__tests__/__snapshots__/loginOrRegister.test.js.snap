// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login or Register Login or Register snapshot 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "backgroundColor": "#F7F9FC",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
>
  <View>
    <Image
      source={
        Object {
          "testUri": "../../../app/assets/img/logo.png",
        }
      }
      style={
        Object {
          "maxHeight": 250,
          "resizeMode": "contain",
          "width": 200,
        }
      }
    />
  </View>
  <ClickButton
    backgroundColor="#5d6fc4"
    onPress={[Function]}
    text="Login"
  />
  <ClickButton
    backgroundColor="#6f347c"
    onPress={[Function]}
    text="SignUp"
  />
</View>
`;
